models.py

```diff
@@ -176,12 +178,19 @@ def creating_session(self):
             
         # set number of transactions back to 0 each round
         self.fc_transactions = 0
-            
+
+
 class Group(BaseGroup):
     pass
     
 
 class Player(BasePlayer):
+    # For detecting bots in this section of the game
+    trading = models.LongStringField(blank=True)
```

Comment:
Only people get Player objects. Bots are only referenceable from session.automated_traders.

This means if you look in Data (in the session webpage), you will only see player data. Bot data gets saved in CSV form to whatever machine is running otree, when the experiment ends

Reply:
The point is every Player represents a person.

Reply:
This was created for detecting bot users of mturk or other online platforms, not for the bots you created

Reply:
ah understood